scripts/plot_performance_breakdown.py

```diff
@@ -25,6 +25,7 @@
 
     sum_parse_duration = 0.0
     sum_sql_translation_duration = 0.0
+    sum_cache_duration = 0.0
```

Comment:
Just a thought for discussion, not directly for this PR: Since we already have the optimization and execution times, could we use those to identify (a) queries for which the runtime of a re-parameterized query strongly deviates from the expected (cached) runtime, and (b) queries for which the execution time dominates the optimization time by far? This information could then be used to block these queries from being cached in their deparameterized form. Best of two worlds?

@j-tr @Bensk1 @Bouncner

Reply:
we already had the idea that we could optimize queries with values from time to time even if they are cached in order to test if the cached plan execution time doesn't deviate too much from the optimized one. that would somehow fit (a).

Reply:
This goes into the direction of (a), but I was more thinking about non-uniformly distributed values. Re-optimizing the query from time to time tackles the problem of outdated statistics, but does not solve the issue of unsafe re-use.

Reply:
unsafe meaning correct but non-optimal?

Reply:
Correct.